packages/shared/src/components/widgets/SocialShare.tsx

```diff
@@ -102,6 +118,19 @@ export const SocialShare = ({
         onClick={() => trackClick(ShareProvider.Email)}
         label="Email"
       />
+      {!isComment &&
```

Comment:
There is an alignment issue
<img width="403" alt="SCR-20230203-of5" src="https://user-images.githubusercontent.com/1488164/216630159-23118083-ff15-4b2b-88ec-0773847d0a4c.png">

Reply:
Oh, good catch! Let me check this quickly! 🚢

Reply:
Context behind the solution: the WhatsApp icon has the correct size but the label below made its total width expand, making it unequal to the email share, which led to making it look like the Squads weren't in the same alignment. I turned it into a grid to make them have the same total width sizes across the board.

Normal size:
<img width="1045" alt="image" src="https://user-images.githubusercontent.com/13744167/216775456-9514c739-128a-4852-84cd-d6c39bb37e6e.png">

Mobile:
<img width="484" alt="image" src="https://user-images.githubusercontent.com/13744167/216775330-99c7b7f0-be25-4c2f-aa78-287bebc6e681.png">

Where there are long labels:
<img width="464" alt="image" src="https://user-images.githubusercontent.com/13744167/216775301-084f9fe0-e45f-4bb3-8dcf-61edd2625463.png">

Let me know what you think!

Reply:
Yeap, looks great, nice work. Just make sure it also works well on narrower phones.